ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor.hs

```diff
@@ -441,18 +441,23 @@ peerSelectionGovernor :: (MonadAsync m, MonadLabelledSTM m, MonadMask m,
                       -> Tracer m (DebugPeerSelection peeraddr)
                       -> Tracer m PeerSelectionCounters
                       -> StdGen
+                      -> Maybe peeraddr -- ^ IPv4 Address
+                      -> Maybe peeraddr -- ^ IPv6 Address
                       -> StrictTVar m (PublicPeerSelectionState peeraddr)
                       -> PeerSelectionActions peeraddr peerconn m
                       -> PeerSelectionPolicy  peeraddr m
                       -> m Void
-peerSelectionGovernor tracer debugTracer countersTracer fuzzRng stateVar actions policy =
+peerSelectionGovernor tracer debugTracer countersTracer fuzzRng mbLocalIPv4
+                      mbLocalIPv6 stateVar actions policy =
     JobPool.withJobPool $ \jobPool -> do
       localPeers <- map (\(target, _) -> (target, 0))
                 <$> atomically (readLocalRootPeers actions)
       peerSelectionGovernorLoop
         tracer
         debugTracer
         countersTracer
+        mbLocalIPv4
```

Comment:
Question - why are these now necessary? are you implying that the `peerSelectionGovernor` needs to know "its own addresses" to operate?